File: nvxapp.server.service/ClientServer-Service/Account/Models/LoginModel.cs
using nvxapp.server.service.ClientServer_Service.ModelsBase;
using nvxapp.server.service.Service.MyTableService.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace nvxapp.server.service.ClientServer_Service.Account.Models
{
    public class LoginInModel
    {
        public string? UserName { get; set; }
        public string? Password { get; set; }
    }

    public class LoginOutModel : ModelResult
    {
        
        public string? Token { get; set; }
        public string? RefreshToken { get; set; }

        public LoginOutModel()
        {
        }
    }

   
}

File: nvxapp.client/src/nvx/ClientServer-Service/Account/Models/login-model.ts
import { ModelResult } from "../../ModelsBase/model-result";




export class LoginInModel {
  public userName: string | null;
  public password: string | null;
}

export class LoginOutModel extends ModelResult {

  public id: string | null;
  public token: string|null;
  //public refreshToken: string | null;

}


